standards/mobile_app_standards.md

```diff
@@ -0,0 +1,135 @@
+Mobile Application Standards
+============================
+
+Scope
+-----
+
+These standards apply to mobile application in-house development and management.
+
+These standards are intended to cover all forms of mobile application: from fully "off-line capable" progressive web applications, through cross-platform apps to fully native apps.
+
+As implied, there are a number of ways of delivering mobile apps. These standards will cover which approach is best and under what circumstances, and aspects that are common to all or many of the approaches. As and when more detail is available on specific approaches, such as Flutter app development, or Progressive Web Apps, such approach-specific standards should form part of their own separate, detailed, documented standard, which, ideally, will simply be a reference to a recognised external standard for that specific technology.
+
+Caveats
+-------
+
+The huge increase in the use of mobile devices means that mobile application development is now of critical interest to the DEFRA family, even more so given the amount of field-work being carried out by workers within the DEFRA family.
+
+This is a fast moving area involving multiple platforms and diverse technology stacks: these standards are therefore early and provisional. As such, these standards often require more background explanation and justification than those for other, more mature or more homogenous, technology stacks. Inevitably, therefore, these standards, for the time being, are likely to be more discursive than those for other areas, and, unavoidably, the line between standard and guidance can often, as yet, not be so sharply drawn.
```

Comment:
Maybe this is true, maybe it's not.

But either way, a bullet-point summary/list of the standards at the top would be helpful for the reader (or at least for this one).

Reply:
Going to try John's "brief standard" followed by justification suggestion.